aws-test/tests/aws_config_rule/variables.tf

```diff
@@ -0,0 +1,86 @@
+
+variable "resource_name" {
+  type        = string
+  default     = "turbot-test-20200125-create-update"
+  description = "Name of the resource used throughout the test."
+}
+
+variable "aws_profile" {
+  type        = string
+  default     = "integration-tests"
```

Comment:
```suggestion
  default     = "default"
```

Reply:
Should we change the default value from `integration-tests` to `default`, I can see in most of the intg test we have `integration-tests` if we change it to `default` in future if we will do automate of running integration test then for it integration test may not pass.

Reply:
@ParthaI In some places we have `integration-tests` and also in some places we have `default` profile in integration test , but we should use `default` and may be later we will need to change the default value from `integration-tests` to `default` for automation to work.